docs/book/src/appendix/keywords.md

```diff
@@ -0,0 +1,88 @@
+# Appendix A: Keywords
+
+The following list contains keywords that are reserved for current or
+future use by the Sway language. As such, they cannot be used as
+identifiers. Identifiers are names of functions, variables,
+parameters, modules, constants, attributes, types or
+traits, ect.
+
+## Keywords Currently in Use
+
+The following is a list of keywords currently in use, with their
+functionality described.
+
+- `as` - rename items in `use` statements, eg `use type::a as alias_name`
+- [`abi`](./docs/book/src/sway-program-types/smart_contracts.md) - defines a smart contract ABI in a syntactcally similar way to traits
```

Comment:
The links are relative to their location in the book, so you can replace "./docs/book/src/" with "../"